dotCMS/src/test/java/com/dotcms/filters/NormalizationFilterTest.java

```diff
@@ -62,18 +63,40 @@ public void test_uri_normalization_invalid_URI() throws IOException, ServletExce
 
         // Each "." segment is simply removed
         originalURI = "./folder/folder1/file.dat";
-        expectedNormalizedURI = "folder/folder1/file.dat";
+        expectedNormalizedURI = "/folder/folder1/file.dat";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "/folder/folder1/file.dat" appears 4 times in this file; the first occurrence is on line 66](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5436097)